<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MesonProjectManager::Internal::MesonBuildSettingsWidget</class>
 <widget class="QWidget" name="MesonProjectManager::Internal::MesonBuildSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QPushButton" name="configureButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Apply Configuration Changes</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="wipeButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Wipes build directory and reconfigures using previous command line options.
Useful if build directory is corrupted or when rebuilding with a newer version of Meson.</string>
     </property>
     <property name="text">
      <string>Wipe Project</string>
     </property>
     <property name="icon">
      <iconset theme=":/utils/images/warning.png"/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="Utils::DetailsWidget" name="container" native="true">
     <widget class="QWidget" name="details" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>371</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Parameters</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="parametersLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="buildDirWidget" native="true"/>
       </item>
       <item>
        <widget class="Utils::FancyLineEdit" name="optionsFilterLineEdit">
         <property name="placeholderText">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Utils::TreeView" name="optionsTreeView">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::DetailsWidget</class>
   <extends>QWidget</extends>
   <header>utils/detailswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>utils/fancylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>Utils::TreeView</class>
   <extends>QTreeView</extends>
   <header>utils/itemviews.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
